sdk/tables/azure-data-tables/azure/data/tables/_serialize.py

```diff
@@ -116,6 +117,7 @@ def _to_entity_guid(value):
 
 
 def _to_entity_int32(value):
+    # TODO: What the heck? below
```

Comment:
lmao did you mean to keep this in

Reply:
Probably not, like to leave myself TODO's throughout that sometimes slip through lol